site/content/docs/user/node-images.md

```diff
@@ -0,0 +1,135 @@
+To specify the build type use the flag `--type`.
+Note however that using `--type=bazel` on Windows or MacOS will not work
+currently due to Kubelet using [CGO] which requires GCC/glibc for linux.
+A workaround may be enabled in the future.
+
+kind will default to using the build type `docker` if none is specified.
+
+```
+kind build node-image --type bazel
+```
+
+Similarly as for the base-image command, you can specify the name and tag of
+the resulting node image using the flag `--image`.
+
+If you previously changed the name and tag of the base image, you can use here
+the flag `--base-image` to specify the name and tag you used.
+
+### Settings for Docker Desktop
+
+If you are building Kubernetes (for example - `kind build node-image`) on MacOS or Windows then you need a minimum of 6GB of RAM
+dedicated to the virtual machine (VM) running the Docker engine. 8GB is recommended.
+
+To change the resource limits for the Docker on Mac, you'll need to open the
+**Preferences** menu.
+<img src="/docs/user/images/docker-pref-1.png"/>
+
+Now, go to the **Advanced** settings page, and change the
+settings there, see [changing Docker's resource limits][Docker resource lims].
+<img src="/docs/user/images/docker-pref-2.png" alt="Setting 8Gb of memory in Docker for Mac" />
+
+
+To change the resource limits for the Docker on Windows, you'll need to right-click the Moby
+icon on the taskbar, and choose "Settings". If you see "Switch to Linux Containers", then you'll need
+to do that first before opening "Settings"
+
+<img src="/docs/user/images/docker-pref-1-win.png"/>
+
+Now, go to the **Advanced** settings page, and change the
+settings there, see [changing Docker's resource limits][Docker resource lims].
+
+<img src="/docs/user/images/docker-pref-build-win.png" alt="Setting 8Gb of memory in Docker for Windows" />
+
+
+You may also try removing any unused data left by the Docker engine - e.g.,
+`docker system prune`.
+
+## Customizing node images
+
+kind images are heavily optimized to be as light as possible, this has the downside that some common programs are missing, such as an editor, ping, ... There is also users that need more specific tools for their clusters or want to embed additional container images to avoid loading once the cluster is up.
+
+It is easy to create a new node image based on a existing one with the additional software, just leveraging the containers capabilities, per example, creating a new Dockerfile that uses the official image as base and adds an step to install the additional software needed. For example:
+
+```
+FROM kindest/node:v1.18.2
+RUN apt-get update && apt-get install -y \
+    iputils-ping \
+    tcpdump \
+    vim \
+ && rm -rf /var/lib/apt/lists/*
+
+COPY my-docker-image.tar /kind
+```
+
+Build the new image and tag it:
+
+```sh
+docker build -t mykindnode:mytag .
+```
+
+And use it to create the cluster:
+
+```sh
+kind create cluster --image mykindnode:mytag
```

Comment:
Nit, I think this line is supposed to be `$ kind create cluster --image mykindnode:mytag`.

Notice the missing `$` at front?

Reply:
a quick look at the docs seems to not use it, so I split it in 2 code sections instead of adding the dollar, seems less confusing.
Good catch